src/lib.rs

```diff
@@ -227,7 +237,7 @@ pub mod pallet {
 		}
 
 		/// Collect whatever portion of your reward are currently vested.
-		#[pallet::weight(0)]
+		#[pallet::weight(T::WeightInfo::claim(0))]
```

Comment:
the weight function generated for `claim` takes a parameter but that parameter is always zero? why

Reply:
never mind, just saw this comment

```
		/// Weight argument is 0 since it depends on how the storage trie is composed
		/// Once we have the number of contributors, we can probably add such a weight here
```

So the complexity parameter is the number of existing contributors?

Reply:
I am observing some constant weight associated to it, but definitely something the benchmarking isnt able to benchmark properly. As said above, I think we should use as a reference the total number of contributors of our crowdloan. In any case, this extrinsic is not callable by a user so its not that important